src/System.Runtime.Intrinsics.X86/src/System/Runtime/Intrinsics/X86/Sse2.cs

```diff
@@ -0,0 +1,545 @@
+// =+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+
+//
+// SSE2.cs
+//
+// A class that implements intrinsic functions to provide access to Intel SSE/SSE2 instructions.
+//
+// =+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+
+
+using System;
+using System.Runtime.Intrinsics;
+
+namespace System.Runtime.Intrinsics.X86
+{
+    /// <summary>
+    /// Provides access to Intel SSE2 hardware instructions via intrinsics
+    /// </summary>
+    /// <para>
+    /// SSE2 class provides access to 128-bit SSE/SSE2 SIMD instructions
+    /// </para>
+
+    public static class Sse2
```

Comment:
Post x64 are there any CPU that don't have both? Does coreclr run on a CPU without Sse2?